models/base/deepcrawl/deepcrawl_proc.sql

```diff
@@ -107,7 +107,7 @@ FROM (
     crawl_date,
     crawl_month,
     crawl_report_month,
-    found_at_sitemap,
+    max(found_at_sitemap) over w1 as found_at_sitemap,
```

Comment:
@Mranfevb92 maybe a good use of string_agg instead of max?  That'd pull in all sitemaps where a URL was found, rather than just one.

Reply:
You are right. Didn't think about that option. Just pushed a commit with that change